client/src/components/SandwichList.js

```diff
@@ -56,7 +56,9 @@ const SandwichList = (props) => {
   }, [])
 
   const toggleFormOnClick = event => {
-    setShowForm(true)
+    if (showForm) {
+      setShowForm(false)
+    } else setShowForm(true)
```

Comment:
this combines a multiline block (the `if` statement) with a one line (the `else` statement)
Either make it all a multiline block, or use a ternary 
multi:
```js
if (showForm) {
      setShowForm(false)
    } else {
      setShowForm(true)
  }
```